pkg/policy/api/selector.go

```diff
@@ -154,6 +145,26 @@ func NewESFromLabels(lbls ...*labels.Label) EndpointSelector {
 	}
 }
 
+// newReservedEndpointSelector returns a selector that matches on all
+// endpoints with the specified reserved label.
+func newReservedEndpointSelector(ID string) EndpointSelector {
+	reservedLabels := labels.NewLabel(ID, "", labels.LabelSourceReserved)
+	return NewESFromLabels(reservedLabels)
+}
+
+var (
+	// WildcardEndpointSelector is a wildcard endpoint selector matching
+	// all endpoints that can be described with labels.
+	WildcardEndpointSelector = NewESFromLabels()
+
+	// ReservedEndpointSelectors are a set of endpoint selectors that only
+	// match endpoints with reserved labels, such as the host or the world.
```

Comment:
Nit: can you change it to end with "that only map reserved labels (e.g. host, or world) to their corresponding endpoint selectors"  ? This more clearly specifies the mapping in the documentation.